config/rootfs/debos/scripts/create_initrd_ramdisk.sh

```diff
@@ -20,6 +20,35 @@ index 4ec926c..ca06c0b 100644
  	# Load ubi with the correct MTD partition and return since fstype
 EOF
 
+patch -p1 << 'EOF'
+--- a/usr/share/initramfs-tools/scripts/nfs-premount/wait_ethernet	1970-01-01 02:00:00.000000000 +0200
++++ b/usr/share/initramfs-tools/scripts/nfs-premount/wait_ethernet	2022-11-25 12:38:46.037498198 +0200
+@@ -0,0 +1,21 @@
++#!/bin/sh
++. /scripts/functions
++
++wait_ethernet() {
++  local netdevwait=60
```

Comment:
I believe `local` is a Bashism, to run with any `sh` it should be dropped.

Reply:
I'm just following style of initramfs tools scripts, i think they expect bash and expect this variable to be local
https://salsa.debian.org/kernel-team/initramfs-tools/-/blob/master/scripts/functions#L325